### PR TITLE
Fix old Plex ValidationFailure message

### PR DESCRIPTION
**Pull request by opeik**
#### Database Migration
**NO**

#### Description
Fixes a really confusing error message I ran into when there isn't a film library in Plex. My guess is this is an old message from when you forked Sonarr.

#### Issues Fixed or Closed by this PR
* Rewords an out of date ValidationFailure exception message that is thrown when there isn't a film library in Plex

### CODE REVIEW

**Comment by galli-leo** [comment]
Could you reword it to say movie library? I think Plex uses movie instead of film.

**Comment by opeik** [comment]
Plex uses "film".

**Comment by galli-leo** [comment]
Says movies for me. I think it's language based (see TV programmes).

**Comment by opeik** [comment]
Oh, that explains it. I changed "film" to "movie" as you requested.

**Comment by galli-leo** [comment]
Thanks for the pr!

**Comment by opeik** [comment]
Not a problem, even if all I did was change a single word in a string 😅.
